src/Components/EmailSummary/index.jsx

```diff
@@ -1,6 +1,12 @@
-function EmailSummary ({name, subject, body, date, read}) {
+import { Link, useSubmit } from "react-router-dom"
+import OpenEmails from "../../Pages/OpenEmailPage"
+
+function EmailSummary ({id, name, subject, body, date, read}) {
+            
+   
     return (
         <div className={`p-4 border-b border-gray-400 border-solid flex justify-between h-25" ${read == 1 ? "bg-header-grey text-white " : ""}`}>
+            <Link to={`/emails/${id}`} onClick={<OpenEmails />}>
```

Comment:
You should move this Link to be the parent element of this component - at the momeny only the text is clickable, but making it the parent will make the whole 'row' clickable